Software/model/model.py

```diff
@@ -1,4 +1,4 @@
-import place
+from Software.model import place
```

Comment:
That looks wrong (and also doesn't match the topic of this PR). Better use the changes from #13 ;-)

Reply:
It seems to me, I often cannot access the packages without adding the upmost package ("Software"). But that might be something with my IDE.

Reply:
That's interesting[tm]. When I try to start the script with `python3 main.py`, it doesn't work with `Software.`

Reply:
Lets review this again after mergin this request - there have been quite some changes